beginners-yoga-course.html

```diff
@@ -0,0 +1,79 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Prema Healing Cleveland</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Signika:300">
+  <link rel="stylesheet" href="main.css">
+  <style type="text/css"></style>
+</head>
+<body>
+<div class="pageWrap">
+
+<!-- HEADER -->
+<div class="gridCell--row gridCell--padded gridCell--row--justifyCenter">
+  <header class="pageHeader">
+    <h1 class="headline pageHeader__headline align--center">
+      <span class="headline__line pageHeader__primary inverted">Prema Healing</span>
+      <span class="headline__line pageHeader__secondary">Cleveland</span>
+    </h1>
+  </header>
+</div>
+
+<div class="gridCell--row program__row gridCell--row--justifyCenter">
+
+  <!-- COL A -->
+  <div class="gridCell--col gridCell--padded program__col align--center">
+    <article class="program program-wide">
+
+      <h2 class="headline inverted uppercase program__title">Beginners Yoga Course</h2>
+
+      <div class="meta uppercase">
+        <p class="meta__item">
+          <span class="unbreakable">A five week course beginning April 13th – May 11th.</span>
```

Comment:
Minor, but any `-` between dates and times should be an en dash. You make it by hitting `option + -` on your keyboard. It's slightly longer than a hyphen. And yes, I'm being serious because I am insane.